android/app/src/dev/res/values/strings.xml

```diff
@@ -0,0 +1,10 @@
+<resources>
+    <string name="app_name">DevAppTemplate</string>
+    <string name="hint_app_url_input">Your local Mendix Studio Pro URL</string>
+    <string name="launch_app_button">Launch</string>
+    <string name="neutral_button">Continue</string>
+    <string name="camera_permission_title">Camera permission</string>
+    <string name="camera_permission_message">To be able to read QR codes you need to grant the Camera permission.</string>
+    <string name="camera_permission_button">Continue</string>
+    <string name="packager_connection_timeout">Please verify that the packager is running.</string>
```

Comment:
Can we make this message as consistent as iOS

Reply:
done